hera_cal/lst_stack/tests/test_calibration.py

```diff
@@ -277,3 +285,47 @@ def test_value_errors(self):
                 run_amplitude_cal=False,
                 run_phase_cal=False,
             )
+
+    def test_relative_phase_calibration(self):
+        uvd = mockuvd.create_uvd_identifiable(
+            integration_time=24 * 3600,
+            ntimes=20,
+            jd_start=2459844.0,
+            ants=[0, 1, 2, 3],
+            time_axis_faster_than_bls=False,
+            pols=["xx", "yy", "xy", "yx"],
+            freqs=mockuvd.PHASEII_FREQS[:100],
+        )
+        stack = LSTStack(uvd)
+        stack.data[1:] = stack.data[0]  # All nights exactly the same
+
+        stack_copy = stack.copy()
+        delta = np.random.uniform(-1, 1, size=(20, 1, 1)) * np.ones((1, 10, 100))
```

Comment:
you're going to need to seed this. The new way to do that is to create `rng = np.random.default_rng(seed=21)` (or whatever seed) and then `rng.uniform()`

Reply:
Changed this